src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -206,17 +230,19 @@ export const PublishingAccordion = ({
       sx={{ bgcolor: 'publishingRequest.light' }}
       elevation={3}
       defaultExpanded={isDraftRegistration || hasPendingTicket || hasMismatchingPublishedStatus}>
-      <AccordionSummary sx={{ fontWeight: 700 }} expandIcon={<ExpandMoreIcon fontSize="large" />}>
-        {registration.status === RegistrationStatus.Unpublished || registration.status === RegistrationStatus.Deleted
-          ? t(`registration.status.${registration.status}`)
-          : t('registration.public_page.publication')}
-        {lastPublishingRequest &&
-          registration.status !== RegistrationStatus.Unpublished &&
-          registration.status !== RegistrationStatus.Deleted &&
-          ` - ${t(`my_page.messages.ticket_types.${lastPublishingRequest.status}`)}`}
-        {!registrationIsValid && !unpublishedOrDeleted && (
+      <AccordionSummary expandIcon={<ExpandMoreIcon fontSize="large" />}>
+        <Typography fontWeight={'bold'} sx={{ flexGrow: '1' }}>
+          {registration.status === RegistrationStatus.Unpublished || registration.status === RegistrationStatus.Deleted
```

Comment:
Ser at det er kopi av gammel kode, så bare ignorer om du vil. (variabelen burde forsåvidt renames til isUnpublishedOrDelete også).
```suggestion
          {unpublishedOrDeleted
```